.github/workflows/build.yaml

```diff
@@ -41,15 +60,37 @@ jobs:
             tar -zxvf copa.tar.gz
             cp copa /usr/local/bin/
 
-      - name: Bats Test
+      - name: Build Copa Action Image
         run: |
-          docker run --net=host --detach --rm --privileged -p 127.0.0.1:8888:8888 --name buildkitd --entrypoint buildkitd moby/buildkit:v${{ env.BUILDKIT_VERSION }} --addr tcp://0.0.0.0:8888
           docker build --build-arg copa_version=${COPA_VERSION} -t copa-action .
+
+      - name: Bats Test - Buildx
+        run: |
           docker run --net=host \
             --mount=type=bind,source=/tmp,target=/data \
             --mount=type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
             --mount=type=bind,source=$GITHUB_OUTPUT,target=$GITHUB_OUTPUT -e GITHUB_OUTPUT \
-            --name=copa-action \
-            copa-action 'docker.io/library/nginx:1.21.6' 'nginx.1.21.6.json' '1.21.6-patched' '10m' 'output.json' 'openvex'
-          docker images
+            --name=copa-action-buildx \
+            copa-action 'docker.io/library/nginx:1.21.6' 'nginx.1.21.6.json' '1.21.6-patched' '10m' 'buildx' 'openvex' 'output.json' 
+          bats --print-output-on-failure ./test/test.bats
+
+      - name: Bats Test - Buildkit Container
```

Comment:
are these going to run sequentially? if so, can we execute in parallel using a matrix